Units/parser-bibtex.r/bib-simple.d/input.bib

```diff
@@ -0,0 +1,27 @@
+% this is a comment that will be ignored
```

Comment:
I would like you to extend the input for testing all kinds.

Reply:
extended both input.bib as well as expected_tags with all bib items in 8f4ee6b